src/ui/Combobox/ComboboxValues.ts

```diff
@@ -54,13 +54,26 @@ export class ComboboxValues implements IComboboxValues {
         { id: `${this.combobox.id}-value-${index}`, className: 'coveo-combobox-value', role: 'option', tabindex: 0 },
         value.element
       ).el;
+      this.highlightCurrentQueryInSearchResults(value.element);
       value.element = elementWrapper;
       fragment.appendChild(value.element);
     });
 
     this.element.appendChild(fragment);
   }
 
+  private highlightCurrentQueryInSearchResults(searchResult: HTMLElement) {
+    if (this.combobox.options.highlightValueClassName) {
+      let regex = new RegExp('(' + this.combobox.element.querySelector('input').value + ')', 'ig');
```

Comment:
In typescript, and ES6, when a variable is not subject to change, we use a `const` instead of a `let`.

Reply:
fixed 👌